File: NapakalakiRuby/lib/cultist.rb
#encoding: utf-8
#Autor: Dunia Rodríguez y Emilio Reguero
module NapakalakiGame
class Cultist

     attr_reader :name, :gainedLevels
  
  def initialize(nam, gainedlevels)
    @name = nam
    @gainedLevels = gainedlevels
  end
  
  def getGainedLevels
    @gainedLevels
  end
end
end

File: Napakalaki/src/NapakalakiGame/Cultist.java
/*
* PDOO Practicas - Napakalaki Java
* @author Emilio Reguero y Dunia Rodríguez 
*/
package NapakalakiGame;


public class Cultist {
    private String name;
    private int gainedLevels;
    
    Cultist(String name, int levels){
        this.name = name;
        this.gainedLevels = levels;
    }
    
    public int getGainedLevels(){
        return gainedLevels;
    }
}
